src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -150,6 +153,31 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         return tagSet;
     }
 
+    /**
+     * Parses delete tag keywords from a String into a Set containing each keyword.
+     *
+     * @param keywords The keywords used to specify which tag to delete
+     * @return The set of string containing the keywords specifying the tags to delete.
+     * @throws ParseException if the keywords given are invalid or are duplicated.
+     */
+    public static Set<String> parseDeleteTagKeywords(String keywords) throws ParseException {
+        requireNonNull(keywords);
+        String trimmedKeywords = keywords.strip();
+        String[] keywordsList = trimmedKeywords.split(" ");
+        final Set<String> keywordSet = new HashSet<>();
+        for (String keyword : keywordsList) {
+            if (!(keyword.equalsIgnoreCase("priority")
+                    || keyword.equalsIgnoreCase("deadline"))) {
+                throw new ParseException(MESSAGE_INVALID_KEYWORDS);
+            }
+            if (keywordSet.contains(keyword)) {
```

Comment:
I was just wondering if it will be better to ignore the duplicate keywords instead of throwing an exception to allow more flexibility